hbt/config/configs_run3.py

```diff
@@ -0,0 +1,803 @@
+
+    cfg.add_shift(name="tautau_trigger_up", id=80, type="shape")
+    cfg.add_shift(name="tautau_trigger_down", id=81, type="shape")
+    add_shift_aliases(cfg, "tautau_trigger", {"tau_trigger_weight": "tau_trigger_weight_tautau_{direction}"})
+    cfg.add_shift(name="etau_trigger_up", id=82, type="shape")
+    cfg.add_shift(name="etau_trigger_down", id=83, type="shape")
+    add_shift_aliases(cfg, "etau_trigger", {"tau_trigger_weight": "tau_trigger_weight_etau_{direction}"})
+    cfg.add_shift(name="mutau_trigger_up", id=84, type="shape")
+    cfg.add_shift(name="mutau_trigger_down", id=85, type="shape")
+    add_shift_aliases(cfg, "mutau_trigger", {"tau_trigger_weight": "tau_trigger_weight_mutau_{direction}"})
+    # no uncertainty for di-tau VBF trigger existing yet
+    # cfg.add_shift(name="mutau_trigger_up", id=86, type="shape")
+    # cfg.add_shift(name="tautauvbf_trigger_down", id=86, type="shape")
+    # add_shift_aliases(cfg, "tautauvbf_trigger", {"tau_trigger_weight": "tau_trigger_weight_tautauvbf_{direction}"})
+
+    cfg.add_shift(name="e_up", id=90, type="shape")
+    cfg.add_shift(name="e_down", id=91, type="shape")
+    add_shift_aliases(cfg, "e", {"electron_weight": "electron_weight_{direction}"})
+
+    cfg.add_shift(name="mu_up", id=100, type="shape")
+    cfg.add_shift(name="mu_down", id=101, type="shape")
+    add_shift_aliases(cfg, "mu", {"muon_weight": "muon_weight_{direction}"})
+
+    btag_uncs = [
+        "hf", "lf",
+        f"hfstats1_{year}", f"hfstats2_{year}",
+        f"lfstats1_{year}", f"lfstats2_{year}",
+        "cferr1", "cferr2",
+    ]
+    for i, unc in enumerate(btag_uncs):
+        cfg.add_shift(name=f"btag_{unc}_up", id=110 + 2 * i, type="shape")
+        cfg.add_shift(name=f"btag_{unc}_down", id=111 + 2 * i, type="shape")
+        add_shift_aliases(
+            cfg,
+            f"btag_{unc}",
+            {
+                "normalized_btag_weight": f"normalized_btag_weight_{unc}_" + "{direction}",
+                "normalized_njet_btag_weight": f"normalized_njet_btag_weight_{unc}_" + "{direction}",
+            },
+        )
+
+    cfg.add_shift(name="pdf_up", id=130, type="shape")
+    cfg.add_shift(name="pdf_down", id=131, type="shape")
+    add_shift_aliases(
+        cfg,
+        "pdf",
+        {
+            "pdf_weight": "pdf_weight_{direction}",
+            "normalized_pdf_weight": "normalized_pdf_weight_{direction}",
+        },
+    )
+
+    cfg.add_shift(name="murmuf_up", id=140, type="shape")
+    cfg.add_shift(name="murmuf_down", id=141, type="shape")
+    add_shift_aliases(
+        cfg,
+        "murmuf",
+        {
+            "murmuf_weight": "murmuf_weight_{direction}",
+            "normalized_murmuf_weight": "normalized_murmuf_weight_{direction}",
+        },
+    )
+
+    # external files
+    json_mirror = "/afs/cern.ch/user/a/anhaddad/public/jsonpog-integration"
+    json_mirror_alt = "/afs/cern.ch/user/a/anhaddad/public/jsonpog_alt"
```

Comment:
`json_mirror = "/afs/cern.ch/user/m/mrieger/public/mirrors/jsonpog-integration-7d9636bf"`
:)

Then we can also change `json_mirror_alt` to `json_mirror` below.